.github/workflows/format.yml

```diff
@@ -6,20 +6,35 @@ jobs:
   black:
     runs-on: ubuntu-latest
     steps:
+      - uses: dorny/paths-filter@v3

```

Comment:
Not sure we need a dedicated Action for filtering. You can use the [`paths` argument](https://github.com/XanaduAI/pennylane.ai-react/blob/staging/.github/workflows/lint-and-test.yml#L3) for the whole workflow.